graylog2-web-interface/src/views/components/Select.jsx

```diff
@@ -11,6 +14,20 @@ const MultiValueRemove = (props) => {
   );
 };
 
+const OverlayInner = ({ children, style }: {children: Node, style?: Object}) => React.Children.map(children,
+  child => React.cloneElement(child, { style: Object.assign({}, style, child.props.style) }));
+
+const menu = (selectRef) => {
+  const defaultMinWidth = 200;
```

Comment:
Should I make this min width optional?
Right now I could not find a use case where the min width of `200px` is not suitable.

Reply:
I think it is fine that way. I would wait for the need.